frontend/src/metabase/embedding-sdk/theme/dynamic-css-vars.ts

```diff
@@ -53,19 +77,8 @@ export function getDynamicCssVariables(theme: MantineTheme) {
         return [cssVar, null];
       }
 
-      let mappedColor = theme.fn.themeColor(operation.source);
-
-      if (operation.lighten) {
-        mappedColor = lighten(mappedColor, operation.lighten);
-      }
-
-      if (operation.darken) {
-        mappedColor = darken(mappedColor, operation.darken);
-      }
-
-      if (operation.alpha) {
-        mappedColor = alpha(mappedColor, operation.alpha);
-      }
+      const baseColor = theme.fn.themeColor(operation.source);
+      const mappedColor = applyColorOperation(baseColor, operation);
```

Comment:
Extracted the previous login into `applyColorOperation` so I can also apply it to the SDK colors. This file is for the custom CSS variables that are not part of SDK colors.